source/request_response.c

```diff
@@ -881,6 +896,13 @@ struct aws_http_message *aws_http2_message_new_from_http1(
         if (aws_http_headers_add(copied_headers, aws_http_header_path, path_cursor)) {
             goto error;
         }
+        AWS_LOGF_TRACE(
+            AWS_LS_HTTP_GENERAL,
+            "Added header \"%.*s %.*s\" to new HTTP/2 header",
```

Comment:
trivial: currently you have the name and value together in the same quotes, I suggest putting something between them. Also, move them to the end of the msg so the arbitrarily long header values fall at the end of the msg

currently this msg will look like:
`Added header ":scheme https" to new HTTP/2 header`

but maybe something like one of these reads better?
`Added header to new HTTP/2 message: ":scheme": "https"`
`Added header to new HTTP/2 message - ":scheme": "https"`
`Added header to new HTTP/2 message - :scheme = https`